OpenOversight/tests/routes/test_auth.py

```diff
@@ -388,6 +377,32 @@ def test_unconfirmed_user_redirected_to_confirm_account(mockdata, client, sessio
         assert b"Please Confirm Your Account" in rv.data
 
 
+def test_disabled_user_cannot_login_or_visit_pages_requiring_auth(
+    mockdata, client, session
+):
+    # All disabled user testing must be done in the same test to avoid
+    # flakiness since is_disabled is being updated
```

Comment:
Could this be avoided by creating a new user to manipulate for each discrete test case? Flakiness due to shared data between test cases is usually more cleanly solved that way rather than putting everything in a single test case.

Reply:
Yes, thanks for the suggestion! Created a separate test user for this test case